.gitignore

```diff
@@ -50,4 +50,8 @@ doc/
 .vscode/settings.json
 
 # RubyMine indexes
-.idea
\ No newline at end of file
+.idea
+
+# Ignore doc generated sites
+documentation/**/site
+documentation/**/.cache
```

Comment:
Needs a final new line character.  This can be automated in your IDE settings:

VSCode:  search for "Final new line" in settings

![image](https://github.com/user-attachments/assets/cce05c57-b61d-4d30-b6e0-eaa0f84cf406)


PyCharm:  In settings, under Editor > General, tick the box called "Ensure every saved file ends with a line break"

![image](https://github.com/user-attachments/assets/65bc81e7-daba-4f3a-8bf1-7689541ce496)

Reply:
Thanks Stuart.